docs/source/tutorials/low-latency.rst

```diff
@@ -120,6 +120,30 @@ Reducing latency when running NN
 In the examples above we were only streaming frames, without doing anything else on the OAK camera. This section will focus
 on how to reduce latency when also running NN model on the OAK.
 
+Resource utilization
+--------------------
+
+Configuring `hardware resources <https://docs.luxonis.com/projects/hardware/en/latest/pages/rvc/rvc2.html#hardware-blocks-and-accelerators>`__
+on RVC will result in lower latency, but also in lower FPS.
+
+By default, NN nodes have run 2 threads and 1 NCE/thread, and it's suggested to compile the model for half of the
+available SHAVEs of the pipeline.
+
+To minimize the latency, we should allocate all resources to the single inference. To get lowest latency (which will result in a bit lower FPS),
```

Comment:
Considerably lower FPS for most models. Full yolo v4 was ever so slightly faster with 1 thread, 10 shaves + 2 NCEs vs 2 threads 5 shaves/1NCE per thread